EIPS/eip-0000.md

```diff
@@ -0,0 +1,494 @@
+---
+eip: 0000
```

Comment:
```suggestion
eip: 7999
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.